Formula/trojan.rb

```diff
@@ -0,0 +1,49 @@
+class Trojan < Formula
+  desc "Unidentifiable mechanism that helps you bypass GFW"
+  homepage "https://trojan-gfw.github.io/trojan/"
+  url "https://github.com/trojan-gfw/trojan/archive/v#{version}.tar.gz"
+  version "1.10.1"
```

Comment:
No, please inline `version` in `url` above, and let brew detect `version` automatically from the `url` string.